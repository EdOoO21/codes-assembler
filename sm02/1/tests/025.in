573 898 727 727 -969 -562 906 785 -376 303 -31 -914 757 951 551 -477 -929 -374 156 -132 -804 205 758 -746 179 406 492 704 -597 49 973 627 935 279 -354 689 881 -630 123 -266 52 -39 74 -788 809 992 707 302
