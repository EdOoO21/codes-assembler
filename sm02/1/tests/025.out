302
707
992
809
-788
74
-39
52
-266
123
-630
881
689
-354
279
935
627
973
49
-597
704
492
406
179
-746
758
205
-804
-132
156
-374
-929
-477
551
951
757
-914
-31
303
-376
785
906
-562
-969
727
727
898
573
